lib/rspec/matchers/dsl27.rb

```diff
@@ -0,0 +1,16 @@
+module RSpec
+  module Matchers
+    # Helper module to include code not parsable on very old rubies
+    module DSL27
```

Comment:
Honestly I'd namespace this under DSL and call it
```suggestion
    module DSL
      module KeywordArgumentSupport
```